drivers/volume/portworx/schedops/k8s-schedops.go

```diff
@@ -479,6 +483,35 @@ func (k *k8sSchedOps) IsPXReadyOnNode(n node.Node) bool {
 	return true
 }
 
+// IsPXEnabled returns true  if px is enabled on given node
+func (k *k8sSchedOps) IsPXEnabled(n node.Node) (bool, error) {
+	isPxEnabled := true
+	t := func() (interface{}, bool, error) {
+		node, err := k8s.Instance().GetNodeByName(n.Name)
+		if err != nil {
+			logrus.Errorf("Failed to get node %v", err)
+			return nil, true, err
+		}
+		return node, false, nil
+	}
+
+	node, err := task.DoRetryWithTimeout(t, 1*time.Minute, 10*time.Second)
+	if err != nil {
+		logrus.Errorf("Failed to get node %v", err)
+		return false, err
+	}
+
+	kubeNode := node.(*corev1.Node)
+	// if node has px/enabled label set to false or node-type public or
+	// has any taints then px is disabled on node
+	if kubeNode.Labels[pxEnabled] == "false" || kubeNode.Labels[nodeType] == "public" || len(kubeNode.Spec.Taints) > 0 {
+		logrus.Infof("[%v] Px Disabled: ", n.Name)
+		isPxEnabled = false
+	}
+	logrus.Infof("PX status on node %v %v", n.Name, isPxEnabled)
```

Comment:
change log to be more explicit.
```
if isPxEnabled {
  logrus.Infof("PX is enabled on node %v.", n.Name)`

} else {
  logrus.Infof("PX is not enabled on node %v. Will be skipped for tests.", n.Name)`
}
```

Reply:
don't need ixPXEnabled here, we can directly return from label/px check